### PR TITLE
Adds Gloves of the Black Silence

### PR DESCRIPTION
**Pull request by scionalu**
## About The Pull Request
Adds the funny gloves (not actual gloves, since I think this should take up one of your ego slots)
Main mechanic is using different weapons to unlock furioso, just a big fuck you attack
Changing weapons gives you a cooldown to prevent people from spamming it, but you can reduce it by attacking

All weapons have a special attack, and this is where this PR might get instantly closed, idk
To trigger the special attack, you need to use SHIFT+CLICK as opposed to z (or whatever you bound it to)
Pressing z here pulls up a menu to change weapons instead (I know you can just make it into UI skill thingy, but I just find pressing z to activate skill annoying af)
Most weapons also have a special gimmick to them, so even if this PR is rejected (probably will), they can still be recycled
Item description need some work tbh...

TLDR;
All base mechanics are coded and somewhat balanced, Sprites are shit, descriptions are minimalistic
There's probably so much more room that can be cleaned up, although I'm not sure when I can work on it again
I hate life

The DPS is between 90-130 depending on the weapon (not counting furioso), just a bit above Grade 1
The sprites are SHIT, I literally just googled the weapons' image and put it to 32x32 lmao
All inhand icons are literally reskins from other egos, they don't even look good nor make sense
I can't sprite, like at all

You can see which weapons you already used in the wheel menu (but again, the sprites are shit)
Each time you change weapon, you get 5 minutes to change again before furioso resets (doesn't increase the timer if you already have furioso unlocked). It's to prevent people from just holding off to it the whole time

## Why It's Good For The Game
I didn't really link it to anywhere, so it's probably just admeme spawn if it even gets in
Also it's literally Black Silence (kinda), love that guy

## Changelog
:cl:
add: Black Silence Gloves
soundadd: Black Silence Weapons, Finger Snap, Index Unlock
imageadd: Shitty sprites that needs to be changed
/:cl:
furioso unlock

### CODE REVIEW

File: code/game/objects/items/ego_weapons/non_abnormality/black_silence.dm [hunk 1]
```diff
@@ -0,0 +1,1033 @@
...
+	icon = 'icons/obj/black_silence_weapons.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons/black_silence_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/black_silence_righthand.dmi'
+	force = 1
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	attack_verb_continuous = list("taps", "pats")
+	attack_verb_simple = list("tap", "pat")
+	hitsound = 'sound/effects/hit_punch.ogg'
+	attribute_requirements = list(
```
Comment (Kitsunemitsu) [inline code/game/objects/items/ego_weapons/non_abnormality/black_silence.dm:17]: Downgrade to 120 all.

Comment (scionalu) [reply code/game/objects/items/ego_weapons/non_abnormality/black_silence.dm:17]: done

**Review by Kitsunemitsu — APPROVED** [review]
Approved.
